### PR TITLE
Game classes

### PR DESCRIPTION
**Pull request by lugard1**
# Description
In this project I:
- [x] Created the branch game-classes.
- [x] Add game class.
- [x] Add author class.
- [x] Fix linter errors.
- [x] Use git-flow.

### CODE REVIEW

File: game/game.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require_relative 'item'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(multiplayer, last_played_at, publish_date)
+    super(publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = Date.iso8601(last_played_at)
```
Comment (mertkantaroglu) [inline game/game.rb]: If you don't need to convert a string in the ISO 8601 format to a Date object, you can skip the Date.iso8601 method. Instead, you can directly assign the last_played_at value to the @last_played_at instance variable if it is already a Date object or any other compatible date representation.

File: game/game.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  end
+
+  def can_be_archived?
+    super || @last_played_at < 2.years.ago
```
Comment (mertkantaroglu) [inline game/game.rb]: I believe this code should be `super && @last_played_at < 2.years.ago`

**Review by mertkantaroglu — CHANGES_REQUESTED** [review]
## CHANGES REQUESTED ♻️ 

Hello @lugard1 👋 

You have done a very good job by implementing requested classes and methods to the project! 👏 

## Highlights

- Descriptive pull request ✔️
- Descriptive commit messages ✔️

On the other hand, there are some minor changes which could make more code cleaner. 

mario-almost there

## Required Changes ♻️ 

Check the comments under the review.

## Optional suggestions

- [ ] Please fix linter errors.

Cheers and Happy coding! 👏👏👏

File: game/game.rb (hunk 1 above)
Comment (lugard1) [reply game/game.rb]: Thanks for the heads up.

File: game/game.rb (hunk 2 above)
Comment (lugard1) [reply game/game.rb]: Thanks.

**Review by mertkantaroglu — APPROVED** [review]
## Status: Approved 🟢 

Hi @lugard1 👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

Congratulations! 🎉 👏 

minions

**Comment by lugard1** [comment]
Thanks for the review.

**Pull request merged**